### PR TITLE
Added a page element with a search form.

### PR DESCRIPTION
**Pull request by IvanBylinovich**
Added a page element with a search form.

### CODE REVIEW

File: src/main/java/by/tms/aviaticket/controller/SearchController.java [hunk 1]
```diff
@@ -26,8 +30,8 @@ public String searchView(){
     }
 
 
-    @PostMapping("search")
-    public String searchView(String from, String to, LocalDate departure, int count, Model model){
+    @GetMapping("search")
```
Comment (SaneaV) [inline src/main/java/by/tms/aviaticket/controller/SearchController.java:33]: It is a good practice to use "/" in mappings. 
From "search" to "/search"

File: src/main/java/by/tms/aviaticket/controller/TicketController.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package by.tms.aviaticket.controller;
+
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("ticket")
```
Comment (SaneaV) [inline src/main/java/by/tms/aviaticket/controller/TicketController.java:9]: It is a good practice to use "/" in mappings.
From "ticket" to "/ticket"

File: src/main/java/by/tms/aviaticket/controller/TicketController.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+public class TicketController {
+
+    @GetMapping//переход на страница для оформления билета
```
Comment (SaneaV) [inline src/main/java/by/tms/aviaticket/controller/TicketController.java:12]: This mapping needs to have value for mapping.

File: src/main/java/by/tms/aviaticket/controller/SearchController.java [hunk 4]
```diff
@@ -26,8 +30,8 @@ public String searchView(){
...
+    public String searchView(@NotBlank @NotNull String from, @NotBlank @NotNull String to, LocalDate departure, @Min(1) @Max(10) int count, Model model){
         List<Flight> flights = searchService.find(from, to, departure, count);
         if (flights.size() == 0){
```
Comment (SaneaV) [inline src/main/java/by/tms/aviaticket/controller/SearchController.java:36]: The flights.isEmpty () entry looks a little better.

File: src/main/webapp/pages/fragments/filter.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <fieldset class="bg-dark text-center text-white" style="height: 50px"  th:fragment="filter">
+        <div class="mb-2" style="float:left;width: 25%; margin-top: 5px" >
+            <input type="text" id="from" class="form-control" placeholder="Пункт вылета">
+        </div>
+        <div class="mb-2" style="float:left;width: 25%; margin-top: 5px">
+            <input type="text" id="to" class="form-control" placeholder="Пункт назначения">
+        </div>
+        <div class="mb-2" style="float:left;width: 20%; margin-top: 5px">
+            <input type="date" id="DepartureDate" class="form-control" placeholder="Дата вылета">
+        </div>
```
Comment (SaneaV) [inline src/main/webapp/pages/fragments/filter.html:19]: It is better to write styles in a special CSS file.

File: src/main/webapp/pages/ticketProcessing.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+        <form action="/???" method="get">
+            <div class="mb-3">
+                <label class="form-label">Firstname</label>
+                <input type="text" class="form-control" id="Firstname">
+            </div>
+            <div class="mb-3">
+                <label class="form-label">Lastname</label>
+                <input type="text" class="form-control" id="Lastname">
+            </div>
+            <div class="mb-2" style="float:left;width: 50%">
```
Comment (SaneaV) [inline src/main/webapp/pages/ticketProcessing.html:22]: It is better to write styles in a special CSS file.

**Review by SaneaV — CHANGES_REQUESTED** [review]

**Review by simonpirko — APPROVED** [review]

**Pull request merged**